andela/wg-silicon
# [Feature #158626113] Rework the landing page

### What does this PR do?
Reworks wger's landing page


### Description of Task to be completed?
Wger's landing page has been reworked to improve its aesthetic value and user experience

### How should this be manually tested?
Visit `https://wg-silicon.herokuapp.com/en/software/features` page. Notice that the UI has changed.

### What are the relevant pivotal tracker stories?
- [Rework landing page](https://www.pivotaltracker.com/story/show/158626113)

### Screenshots
- __Before Reworking the landing page__
<img width="1388" alt="screen shot 2018-07-27 at 12 40 01" src="https://user-images.githubusercontent.com/29580769/43314255-152acf0c-919b-11e8-8df7-0d29bcdc9e12.png">

<img width="1373" alt="screen shot 2018-07-27 at 12 40 17" src="https://user-images.githubusercontent.com/29580769/43314284-2633961c-919b-11e8-8bd1-9c0a8ff130c5.png">

- __After Reworking the landing page__
<img width="1417" alt="screen shot 2018-07-27 at 12 39 10" src="https://user-images.githubusercontent.com/29580769/43314303-346d704a-919b-11e8-9a31-7e6b27a0ac48.png">

<img width="1399" alt="screen shot 2018-07-27 at 12 39 33" src="https://user-images.githubusercontent.com/29580769/43314319-40f214d8-919b-11e8-8293-161003b9628b.png">
